generative_models/VAEGAN-PYTORCH/README.md

```diff
@@ -0,0 +1,189 @@
+# Paper
+
+* **Title**: Autoencoding beyond pixels using a learned similarity metric
+* **Authors**: Anders Boesen Lindbo Larsen,Søren Kaae Sønderby,Hugo Larochelle,Ole Winther
+* **Link**: https://arxiv.org/pdf/1512.09300.pdf
+* **Tags**: Neural Network, Generative Networks, GANs
+* **Year**: 2016
+
+### Usage
+```bash
+$ python3 main.py 
+```
```

Comment:
reformat this as https://github.com/pclubiitk/model-zoo/blob/master/generative_models/cGAN_PyTorch/README.md